Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM centos:centos7
+LABEL maintainer="Nik Hodgkinson <11xor6@gmail.com>"
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.11.0
+RUN yum -y install epel-release && \
+    yum -y update && \
+    yum -y install java-1.11.0-openjdk-devel python-pip && \
+    yum clean all && \
+    rm -rf /var/cache/yum && \
+    pip install --upgrade pip && \
+    pip install dumb-init
+
+ARG PRESTO_VERSION
+RUN mkdir -p /data/presto && \
+    mkdir -p /usr/lib/presto/etc/catalog && \
+    cd /tmp && \
+    curl http://central.maven.org/maven2/io/prestosql/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz | tar xz && \
```

Comment:
use binaries that are in repo

Reply:
Done.

Reply:
Additionally, we should publish a `Dockerfile` for users which does retrieve its artifacts from Maven central. I'm open to suggestions about how we should supply that particular `Dockerfile`.